### PR TITLE
Use tilde (~) to resolve the home directory (vagrant case)

### PR DESCRIPTION
**Pull request by a4099181**
The environment variables `$env:HOMEDRIVE` and `$env:HOMEPATH` are empty
when vagrant executes `KubeCluster.ps1 -install`.
It makes _SSH key_ is not found even when it is correctly provisioned.
Script asks to generate _SSH key_ but it shouldn't do it.

### CODE REVIEW

**Comment by neolit123** [comment]
> The environment variables $env:HOMEDRIVE and $env:HOMEPATH are empty

i'm pretty sure you can set these env. variables in vagrant?

File: kubeadm/v1.15.0/KubeCluster.ps1 [hunk 1]
```diff
@@ -193,7 +193,7 @@ if ($install.IsPresent)
     InstallKubernetesBinaries -Destination  $Global:BaseDir -Source $Global:ClusterConfiguration.Kubernetes.Source
     DownloadCniBinaries -NetworkMode $Global:NetworkMode -CniPath $(GetCniPath)
 
-    if (!(Test-Path $env:HOMEDRIVE/$env:HOMEPATH/.ssh/id_rsa.pub))
+    if (!(Test-Path ~/.ssh/id_rsa.pub))
```
Comment (neolit123) [inline kubeadm/v1.15.0/KubeCluster.ps1]: does `~` work in PowerShell?

**Comment by neolit123** [comment]
>  GitHub didn't allow me to assign the following users: ksubrmnn.

@ksubrmnn given you are already a member of the kubernetes org, you are eligible for kubernetes-sigs too. if you file an  issue here https://github.com/kubernetes/org/issues you can become a member.

File: kubeadm/v1.15.0/KubeCluster.ps1 (hunk 1 above)
Comment (neolit123) [reply kubeadm/v1.15.0/KubeCluster.ps1]: EDIT: looks like it does.

Comment (a4099181) [reply kubeadm/v1.15.0/KubeCluster.ps1]: Yes, it works well. I tested it.

**Comment by a4099181** [comment]
> > The environment variables $env:HOMEDRIVE and $env:HOMEPATH are empty
> 
> i'm pretty sure you can set these env. variables in vagrant?

Yes, I can. But in my opinion, this is not the best option to fix it in Vagrantfile. The key problem of that case will not disappear and still, sometimes the script will stop automation.

**Comment by a4099181** [comment]
I think _Active Directory_ is an important case because `$env:HOMEDRIVE` and `$env:HOMEPATH` may point to a remote user profile when a user is logged into a domain (AD).
In this case `~` and `$env:USERPROFILE` still points to a local user profile.
If so, `~` may be used as a fallback for the case when `$env:HOMEDRIVE` and `$env:HOMEPATH` are empty.
This is my alternative solution implemented in 2nd commit.

**Comment by ksubrmnn** [comment]
@neolit123 will take a look at this next week. And I'll join Kubernetes-sigs as well

**Comment by ksubrmnn** [comment]
lgtm

**Pull request merged**